rct229/ruleset_functions/compare_schedule_test.py

```diff
@@ -36,6 +36,23 @@ def test__Test_Schedule_Compare_Success_2():
     )
 
 
+def test__Test_Schedule_Compare_Success_3():
```

Comment:
I looks like the `compare_schedules` function is being tested. So I think the name of this test should be something like `test__compare_schedules__not_identical`.

Reply:
Great point! Let me get the function name fixed. I'll also fix the other function names as well.